nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/model/Property.java

```diff
@@ -493,6 +493,23 @@
      */
     Property get(int index) throws PropertyNotFoundException;
 
+    /**
+     * Sets the child property having the given name.
+     * <p>
+     * The given name should be the full name (i.e. prefixed name if any prefix exists).
+     * <p>
+     * If a non prefixed name is given, the first child property having the given local name will be returned.
```

Comment:
i think  If a **non** prefixed  -> If a **not** prefixed

Reply:
I don't know what is the best, I've just copied this line from the method above and I prefer to be consistent.